<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmNewPaciente</class>
 <widget class="QDialog" name="FrmNewPaciente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuevo Paciente</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbH">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>60</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Hembra</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbM">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>60</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Macho</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Dueño:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDuenio">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>90</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>160</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha de Nacimiento:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Genero :</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="deFecha">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>200</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sintomas :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSintomas">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdRegistrar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCerrar">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>300</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nro Hermanos :</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbHermanos">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>124</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>DNI Dueño:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDni">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>124</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtNombre</tabstop>
  <tabstop>rbH</tabstop>
  <tabstop>rbM</tabstop>
  <tabstop>txtDuenio</tabstop>
  <tabstop>txtDni</tabstop>
  <tabstop>deFecha</tabstop>
  <tabstop>txtSintomas</tabstop>
  <tabstop>sbHermanos</tabstop>
  <tabstop>cmdRegistrar</tabstop>
  <tabstop>cmdCerrar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
